### PR TITLE
cli: changed help message (-h) to start with capital letter

### LINKED ISSUE #1064: Use consistent help messages style everywhere
**Option/argument:**
   
   start lower case (this way we don't have to fix help thing), no dot after

**Command description:**

  Upper case, dots, sentences.

### PR DESCRIPTION
**Pull request by Hiyorimi**
Fixes #1064

### CODE REVIEW

**Comment by efiop** [comment]
@Hiyorimi Could you please describe what is going on here? Not quite sure how this fixes 1064.

**Comment by Hiyorimi** [comment]
@efiop I will clear things up in the issue

File: dvc/cli.py [hunk 1]
```diff
@@ -144,6 +144,17 @@ def parse_args(argv=None):
         description=desc,
         parents=[parent_parser],
         formatter_class=argparse.RawTextHelpFormatter,
+        add_help=False,
+    )
+
+    # NOTE: adding this argument in get_parent_parser() either in
```
Comment (efiop) [inline dvc/cli.py]: Let's clarify in the comment that we are only doing this to make the message start from a capital letter.

**Comment by efiop** [comment]
@Hiyorimi Thanks for the clarification! Let's adjust the comment, PR/commit name and description to reflect that we are only doing this to get `-h` message start from a capital letter.

**Comment by Hiyorimi** [comment]
@efiop done. Thought that shorter commit message is better, hence it is not that grammatically great.

File: dvc/cli.py (hunk 1 above)
Comment (efiop) [reply dvc/cli.py]: @Hiyorimi Let's add the comment on why we are doing this. It is not clear from the code. :slightly_smiling_face:

**Comment by efiop** [comment]
Thanks @Hiyorimi ! Looks like you've missed https://github.com/iterative/dvc/pull/2591#discussion_r334873661 . :slightly_smiling_face: 

For the record: checked our docs and there are no old `show this help message and exit` strings in them, as we've replaced them with a hand-written description there.

File: dvc/cli.py (hunk 1 above)
Comment (Hiyorimi) [reply dvc/cli.py]: Sorry, missed that. Now fixed that.

**Pull request merged**